### PR TITLE
DOIs for 2018

### LINKED ISSUE #435: W18 DOI assignation
This is a placeholder issue for @knmnyn to assign DOIs to 2018 workshops.  

Crossreferencing #158.

### PR DESCRIPTION
**Pull request by knmnyn**
Closes #435 and #436 .  Related to PR on add_dois.py (yet to be completed)

### CODE REVIEW

File: data/xml/D18.xml [hunk 1]
```diff
@@ -25,6 +25,7 @@
       <url>D18-1001</url>
       <abstract>This article deals with adversarial attacks towards deep learning systems for Natural Language Processing (NLP), in the context of privacy protection. We study a specific type of attack: an attacker eavesdrops on the hidden representations of a neural text classifier and tries to recover information about the input text. Such scenario may arise in situations when the computation of a neural network is shared across multiple devices, e.g. some hidden representation is computed by a user’s device and sent to a cloud-based model. We measure the privacy of a hidden representation by the ability of an attacker to predict accurately specific private information from it and characterize the tradeoff between the privacy and the utility of neural representations. Finally, we propose several defense methods based on modified training objectives and show that they improve the privacy of neural representations.</abstract>
       <video href="https://vimeo.com/305202770" tag="video"/>
+      <doi>10.18653/v1/D18-0101</doi>
     </paper>
```
Comment (mjpost) [inline data/xml/D18.xml]: This one isn't correct—it should be D18-1001. I wonder if you used the old buggy `construct_anthology_url`?

**Comment by knmnyn** [comment]
Hi @mjpost, it seems that all of the conference DOIs were wrong after I
finagled the length part of the DOIs in the `utils.py`.  I'm regenerating
these for [PDK]18.xml and will recommit soon.

**Comment by mjpost** [comment]
Hi Min—Can you confirm that these latest DOI insertions use the code that first verifies that the URL exists? If so, I'll approve!

**Comment by knmnyn** [comment]
Yes, it does.  It's much slower this way but it does check.  Thanks!

**Pull request merged**